core/extension/gdextension_compat_hashes.cpp

```diff
@@ -698,14 +698,14 @@ void GDExtensionCompatHashes::initialize() {
 		{ "framebuffer_create", 1884747791, 3284231055 },
 		{ "framebuffer_create_multipass", 452534725, 1750306695 },
 		{ "framebuffer_create_empty", 382373098, 3058360618 },
-		{ "vertex_buffer_create", 3491282828, 3410049843 },
+		{ "vertex_buffer_create", 3491282828, 76995708 },
 		{ "vertex_array_create", 3137892244, 3799816279 },
-		{ "index_buffer_create", 975915977, 3935920523 },
+		{ "index_buffer_create", 975915977, 2360653086 },
 		{ "shader_compile_spirv_from_source", 3459523685, 1178973306 },
 		{ "shader_compile_binary_from_spirv", 1395027180, 134910450 },
 		{ "shader_create_from_spirv", 3297482566, 342949005 },
 		{ "shader_create_from_bytecode", 2078349841, 1687031350 },
-		{ "uniform_buffer_create", 1453158401, 34556762 },
+		{ "uniform_buffer_create", 1453158401, 3410049843 },
```

Comment:
These changes should be removed. Registering the compatibility methods should be enough.

Reply:
Makes sense, I did suspect it when I saw the the file history didn't showcase any other hash changes